web/src/stream_settings_data.ts

```diff
@@ -28,6 +32,11 @@ export type SettingsSubscription = StreamSubscription & {
 };
 
 export function get_sub_for_settings(sub: StreamSubscription): SettingsSubscription {
+    const creator = stream_data.maybe_get_creator_details(sub.creator_id);
+    const creator_info = {
+        creator,
+        is_active: creator ? people.is_person_active(creator.user_id) : false,
+    };
```

Comment:
Might be better to just add the `is_active` field on the `creator` object itself and not have this `creator_info` object.